src/v2/Apps/Conversation/Utils/usePoll.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from "react"
+
+type IntervalFunction = () => unknown | void
+
+export default function usePoll(
```

Comment:
Less horrible `setInterval` hook. Ideally we can replace this with subscriptions in the future, or switch to using relay hooks as an intermediary step.

Reply:
We should [avoid default exports](https://github.com/artsy/force/blob/master/docs/best_practices.md#naming-imports-and-exports) as they're not statically analyzable like named exports are (a named export can be one thing and one thing only when imported, unless deliberately aliased which takes extra effort). 

The reason is default exports often lead to drift because devs will import them and assign a variety of names to the variable.

Reply:
The avoiding default exports RFC was mine, so I should've known better 🙈